Digital clock/readme.md

```diff
@@ -0,0 +1,26 @@
+**Digital clock**
+
+**GOAL**
+
+Show time in laptop
+
+**DESCRIPTION**
+
+IT is digital clock.
+
+**WHAT I HAD DONE**
+
+Write a python code fetch data from computer hardware and show on screen.
+
+**LIBRARIES NEEDED**
+
+ tkinter
+
+**DEMONSTRATION**
+
```

Comment:
Please add the demonstration images here,
syntax for adding images, `![image name](image URL fetched from the Images folder)`